runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java

```diff
@@ -270,24 +271,11 @@ public NettyClientRouter(String host, Integer port, Boolean tls,
         counterAsyncOpSent = metrics.counter(pfx + "async-op-sent");
 
         if (tls) {
-            sslContext =
-                    TlsUtils.enableTls(TlsUtils.SslContextType.CLIENT_CONTEXT,
-                            keyStore, e -> {
-                                throw new RuntimeException("Could not read the key store "
-                                        + "password file: " + e.getClass().getSimpleName(), e);
-                            },
-                            ksPasswordFile, e -> {
-                                throw new RuntimeException("Could not load keys from the key "
-                                        + "store: " + e.getClass().getSimpleName(), e);
-                            },
-                            trustStore, e -> {
-                                throw new RuntimeException("Could not read the trust store "
-                                        + "password file: " + e.getClass().getSimpleName(), e);
-                            },
-                            tsPasswordFile, e -> {
-                                throw new RuntimeException("Could not load keys from the trust "
-                                        + "store: " + e.getClass().getSimpleName(), e);
-                            });
+            try {
+                sslContext = TlsUtils.enableTls(false, keyStore, ksPasswordFile, trustStore, tsPasswordFile);
+            } catch (SSLException e) {
+                throw new RuntimeException(e);
```

Comment:
Do we need to wrap this in a RuntimeException?
I suppose it was wrapped earlier as lambdas cannot surface any other exceptions.

Reply:
Not my call, I'm leaving it in as it's the previous behaviour.